src/relation_engine_server/api.py

```diff
@@ -23,12 +23,20 @@ def run_query():
     Run a stored view as a query against the database.
     Auth: only kbase users (any role)
     """
-    auth.require_auth_token(roles=[])
+    json_body = flask.request.json or {}
+    if 'query' in json_body:
```

Comment:
This does have the side effect of prohibiting "query" as a view variable. Probably fine but we might want to document that in the views readme